pkg/controlplane/authz.go

```diff
@@ -129,9 +129,7 @@ func (cp *Instance) AuthorizeEgress(req *EgressAuthorizationRequest) (*EgressAut
 }
 
 // AuthorizeIngress authorizes a request for accessing an exported service.
-func (cp *Instance) AuthorizeIngress(
-	req *IngressAuthorizationRequest,
-	peer string,
+func (cp *Instance) AuthorizeIngress(req *IngressAuthorizationRequest, peer string,
```

Comment:
Try to break multiline parameter lists on parameters instead of adding a trailing comma?
```suggestion
func (cp *Instance) AuthorizeIngress(req *IngressAuthorizationRequest, 
  peer string) (*IngressAuthorizationResponse, error) {
```

Reply:
Doesn't work. WIth your suggestion getting format error:  File is not `gofumpt`-ed (gofumpt) which bring back to the current version.